packages/block-library/src/post-title/style.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-post-title > a {
```

Comment:
Can you change this to `.wp-block-post-title a`? That will reduce the specificity ever so slightly, and I would not think anyone nest A's anyway.